src/customers/mapper/customer.mapper.ts

```diff
@@ -0,0 +1,18 @@
+import { Customer } from "../domain/customer.entity";
+import { CustomerDto } from "../dto/customer.dto";
+
+
+export class CustomerMapper {
```

Comment:
**suggestion(non-blocking):** in JS, you can just export function directly if you don't need a class for a specific reason (compared to other languages like Java).
```typescript
export function toCustomerDto(customer: Customer): CustomerDto {
      const { id, firstName, lastName, username, email, password} = customer;

      return { id, firstName, lastName, username, email, password };
}
```

Reply:
Done the suggested modifications.
Here I thought that we should use classes for the mappers as a convention.

Reply:
I tagged it as non-blocking, since its not that relevant and more subjective in terms of the code-review.